acceptance/acceptance_test.go

```diff
@@ -1131,6 +1132,9 @@ func testAcceptance(t *testing.T, when spec.G, it spec.S, packFixturesDir, packP
 
 			packageName := createPackageLocally(aggregatePackageToml)
 			assertImageExistsLocally(packageName)
+
+			defer h.DockerRmi(dockerCli, nestedPackageName)
```

Comment:
Minor: Sorry if I wasn't clear enough. 

This line should be right after line 1127, where the `nestedPackageName` was created.

The line below should be right after line 1133, where `packageName` was created.

This would ensure that defer is registered as early as possible and execute even if other assertions fail.

Reply:
Not a problem.  I found additional images like the one above and am in the process of fixing them.